### ISSUE TITLE
[Bug] handcraft  mod

### ISSUE THREAD

**Issue by MeiyuanaQ** [Traben-0/Entity_Model_Features#62]
When using this mod, all the furniture blocks added in the module handcraft will cause a large frame drop
handcraft:https://www.curseforge.com/minecraft/mc-mods/handcrafted

**Comment by milkembers** [comment]
i don't think this is this mod - before i used this mod, handcrafted still caused large frame drops. there's an open issue on handcrafted's github for it.
